src/lib/coerce.js

```diff
@@ -175,6 +177,18 @@ exports.valObjects = {
                 return;
             }
             propOut.set(dflt);
+        },
+        validateFunction: function(v, opts) {
+            var dflt = opts.dflt,
+                dlen = dflt.length;
+
+            if(v === dflt) return true;
+            if(typeof v !== 'string') return false;
+            if(v.substr(0, dlen) === dflt && idRegex.test(v.substr(dlen))) {
+                return true;
+            }
+
+            return false;
```

Comment:
@alexcjohnson I chose to write a `validateFunction` for the `'subplotid'` val type (only `'subplotid'` and `'info_array'` now have `validateFunction`).

This felt like (1) less of a hassle than adding a `base` option and (2) less hacky than trying to _patch_ into `Lib.validate`. I hope you're ok with :arrow_double_up: .